Flow/question-three.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="../CSS/typography.css" />
+    <link rel="stylesheet" href="../CSS/Global-styles.css" />
+    <link rel="stylesheet" href="../CSS/Homepage.css" />
+    <link rel="stylesheet" href="../CSS/Normalize.css">
+    <title>Question-Three</title>
+</head>
+<body>
+    <header class="header">
+        <h1 class="header__title">Title</h1>
+    </header>
+
+    <main>
+        <form id="form__list" class="form__list">
+            <fieldset class="form__container">
+                <legend class="title__question">Which of these variables never changes value?</legend>
+                <p  class="form__range">
+                    <input
+                        id="answerSlider"
+                        class="slider"
+                        type="range"
+                        min="1"
+                        max="3"
+                        step="1"
+                        name="query3"
+                        value="1"
+                    />
+                </p>
+
+                <p class="slider__item">
+                    <label for="Var">Var</label>
```

Comment:
dont repeat yourself

Reply:
Hi, I fixed the don't repeat yourself, I changed the for attribute.